_includes/components/tiles.njk

```diff
@@ -0,0 +1,22 @@
+<div class="tiles-block">
+	<div class="container container--wide container--flex">
+		<div class="tiles-block__tile">
+			<hr>
+			<h4>Leadership Programme</h4>
+			<p>Our flagship leadership programme brings together 100 of the most senior people across the public sector each year.</p>
+			<p><a href="{{ '/our-programme/' | url }}">Read more about our programme ></a><p>
+		</div>
+		<div class="tiles-block__tile">
+			<hr>
+			<h4>Network of peers</h4>
```

Comment:
Just double checking if h4s are the right choice here, though I know we're missing other content on this page right now

Reply:
Whether or not to use h4s here was something I was considering too. My logic was that it made sense to use a h4 as semantically, these are component headers. Element styling only went up to h3, so I wrote h4 styles for instances of slightly larger bold text in this style. What do you think?

Reply:
I've usully handled this by removing styling from the heading elements and into class names, then you can pick whatever heading level makes sense in the context of the page, and style it however you want.

As an example, gov.uk does that same thing: https://design-system.service.gov.uk/styles/typography/